packages/application-generic/src/usecases/compile-template/compile-template.usecase.ts

```diff
@@ -50,6 +50,21 @@ Handlebars.registerHelper(
   }
 );
 
+Handlebars.registerHelper(
+  HandlebarHelpersEnum.UNIQUE,
+  function (array, property) {
+    if (!Array.isArray(array)) return '';
+
+    return array
+      .map((item) => {
+        if (item[property]) {
```

Comment:
@jainpawan21 wdyt of also making it work for unique values of an array of strings?